src/assessments/assessment-default-message-generator.scss

```diff
@@ -3,5 +3,7 @@
 .no-failure-view {
     font-size: 15px;
     font-weight: normal;
-    padding: 15px 0 15px 15px;
+    padding-top: 15px;
```

Comment:
I verified that `padding-right: 0` is not needed.

Reply:
I agree!